texastribune/donations-app
# Add in google drive gem to load in faq from google sheets

#### What's this PR do?

Adds in the middleman_google-drive gem to begin loading in info from a google spreadsheet. This starts by loading in FAQ questions and answers.
#### Why are we doing this? How does it help us?

Step toward having membership be able to update info in the Support Us app without a code push or anything from tech.
#### How should this be manually tested?

First, you'll need to have the google doc shared with you, so look in the wiki for info on that.

Then `bundle update` to update your Ruby gems.

Run the site locally:
`bundle exec middleman`

You should see on the command line that it's loading data from the google doc.

Look at /faq.html locally, and see that the questions/answers loaded in match what's in the google doc.
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?
#### What are the relevant tickets?

Github issue: https://github.com/texastribune/donations-app/issues/5
#### Next steps?

Add in ability to enter contact info and more through google docs.
#### Smells?
#### TODOs:
- [x] Share google doc with membership and fill them in on how to update
- [x] Remove yml files that are no longer needed
- [x] Update to load in contacts from google doc too
- [x] Also move info, support, and tagline into google docs
- [x] Look into ways to auto-update on save of doc without a deploy
#### Has the relevant documentation/wiki been updated?

Updated README. Will update wiki to reflect this update, for tech and for membership.
#### Technical debt note

Same/less
